eclipsesource/cdtcloud-deploymentserver
# License header new

## Changes Proposed:


- added license headers
- added eslint to deployment-server-ui and cdtcloud

## Issues Addressed:


- closes #80 

## Tests Performed:


-


## How to Test the Changes:


1. Change License header in any file
2. in cdtcloud-deploymentserver: 'yarn run lint'
3. change should show as an error - 'yarn run lint --fix' should fix the change to match the correct license header again

## Known Issues and TODO List:


- [ ] additional lint fixes in deployment-server-ui and cdtcloud (some lint rules have been switched off)

## How to Test PRs
1) First fetch the Pull-Request from the main repo, replacing `xx` with the ID of the PR:\
`git fetch git@github.com:eclipsesource/cdtcloud-deploymentserver.git pull/xx/head:pr_xx`
2) Checkout the Pull-Request, again replacing `xx` with the PR-ID:\
`git checkout pr_xx`
3) Perform the necessary tests for the PR
